openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -147,3 +155,15 @@ export function initPreviewButton() {
         });
     });
 }
+
+export function initWebsiteTranslationOptions() {
+    $(document).ready(function () {
+        $('#locale-options li a').on('click', function () {
+            event.preventDefault();
+            const locale = $(this).data('lang-id');
+            setValueInCookie('HTTP_LANG', locale);
+            location.reload();
+        });
+    });
```

Comment:
- Don't need `document.ready`; that's already happened at this point

```suggestion
    $('#locale-options li a').on('click', function (event) {
        event.preventDefault();
        const locale = $(this).data('lang-id');
        setValueInCookie('HTTP_LANG', locale);
        location.reload();
    });
```